# Introducing the Outbreak Data Analysis Platform (ODAP) 

UK science has been at the forefront of the scientific endeavours to understand the biology of the SARS-CoV-2 virus and to find better ways to treat and prevent COVID-19 disease.  

Important UK based research studies have generated critical scientific data that has enabled scientists and doctors to make important discoveries including: 

1. Tracking the spread of the SARS-CoV-2 viruses by sequencing the ‘DNA’ of the virus. This has been done for more than a million COVID-19 cases – and enable scientists to identify emergence and spread of ‘variants’ such as the Alpha, Delta and now Omicron variant.  

2. Understanding COVID-19 disease and then assessing the effects of variants and vaccination on disease severity. This is done by doctors and nurses in intensive care units (ICUs) in hospitals across making detailed records of the clinical characterisation of COVID-19 disease (e.g. patients condition, treatments, progression)  

3. Understanding how differences in people’s DNA can affect if they will develop severe COVID-19 disease. This was done by sequencing the DNA of people who were in intensive care with severe COVID-19 disease. This has identified 23 differences in our DNA that can lead to greater severity, scientists will now use this understanding to develop new treatments for COVID-19. 

The findings made by analysing the data from these three studies have all alone brought benefits to public health and patients, including being used by government to help manage the pandemic. 

The joined-up nature of health care data in the UK National Health Service (NHS) makes the UK one of the very few places in the world were this can be done in big enough numbers to make new discoveries possible. It offers the opportunity for scientists to make new discoveries to help treat COVID-19 patients and to improve the pandemic response.  

The Outbreak Data Analysis Platform (ODAP) partnership has now been set up by an overarching grouping of UK data generators and provides oversight, strategic guidance and governance to the ODAP platform. This brings together the data from these three studies (and others) into one trusted research environment (TRE) supported by a high-performance computing space to build on this strength of the UK National Health Service and allow scientists to analyse the together all together for the first time. The data will be held securely and anonymously, and access will be restricted to only accredited scientists who will be able to access and use the data – after going through a ‘five safes’ (URL) application process. 

For the long term - being able to analysis different types of pandemic health care data rapidly in a single location will enable the UK to be better prepared for future pandemics and to routinely conduct infectious disease research (e.g. flu, sepsis and antibiotic resistant bacteria (aka ‘Superbugs’). 

The UKRI-supported Outbreak Data Analysis Platform (ODAP) ([isaric4c.net/ap](https://isaric4c.net/ap)) is based at the University of Edinburgh and EPCC/eDRIS and runs a trusted research environment (TRE) that securely holds unique aggregation of UK sovereign data assets. This includs the complete data resources of the ISARIC4C/CO-CIN (clinical characterisation of COVID-19 patients), GenOMICC (host genetics of severe COVID-19 patients), PHOSP (follow up of COVID-19 patients), HEAL COVID (trial), and UK-CIC (immunology of COVID-19 patients) studies, together with COG-UK (SARS-CoV-2 viral sequence from ~15% of all cases) data from England and Scotland, and linkage to the RECOVERY trial, NHS clinical records, national vaccination records, critical care audit data (SICSAG) and mortality records. Linkage to ICNARC CMP has already been agreed.
 
The ODAP will allow academic and public health researchers access to the data following a ‘five safes’ (http://www.fivesafes.org) research application and governance process.

Once granted access researchers for a defined research project they can then make use of multiple datasets in the ODAP, handling sensitive data in the trusted research environment and then exporting deidentified (e.g. anonymous) summary data over to a high-performance computing space where it can be analysed along with non-sensitive data such as human and viral genomes. 
